### DISCUSSION TITLE
How to destroy the kaplay instance?

### DISCUSSION THREAD

**Discussion by BananaAcid** [kaplayjs/kaplay#271] [Q&A] [answered]
Using Vue3, I want to destroy the kaplay instance, when unmounting a view, or at specific cases manually.

In both cases: globally with `kaplay();` and locally with `const K = kaplay({global: false});`

What would be the best way to do so?

**Comment by lajbel** [comment, marked as answer]
https://kaplayjs.com/doc/ctx/quit/

Would work for both cases

By the way, you can't run more than 1 kaplay instance per page in latest alphas.

**Reply by BananaAcid** [reply]
Thanks, that helps.

is there a specific reason, it won’t work? 
I thought using the init options canvas and global:false schools work.

**Reply by lajbel** [reply]
It doesn't work because there's some variables that are not resetted. I opened an issue to adress that #273
